ci/spelling-config.json

```
@@ -17,6 +17,7 @@
         "HIPAA",
         "CODEOWNERS",
         "explainability",
-        "APAC"
+        "APAC",
+        "darkmodesafe"
```

i think the better fix for this is for cpell to ignore that line instead.

```suggestion
        "APAC",
```